core/src/net/sf/openrocket/database/AsynchronousDatabaseLoader.java

```diff
@@ -12,6 +12,8 @@
 	private volatile boolean startedLoading = false;
 	private volatile boolean endedLoading = false;
 	private volatile boolean inUse = false;
+
+	private boolean forceSynchronousLoading = true;
```

Comment:
Should we bother with fixing/updating the Java docs around changes made in this file?